docs/api.md

```
@@ -0,0 +1,320 @@
+# API v1 — curl Reference
+
+Base URL: `http://localhost/api/v1`
+
+Replace `TOKEN` with your personal access token from `/settings/api-tokens`.
+
+---
+
+## Authentication
+
+All requests require:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)